### ISSUE TITLE
Part of `Options` is not optional anymore

### ISSUE THREAD

**Issue by Hatko** [includable/react-native-map-link#50]
According to the `Basic usage` section from *REDME*, `dialogTitle`, `dialogMessage` and `cancelText` are optional. But after update to "^2.1.2" they don't seem anymore:

`index.d.ts`
```
interface Options {
    latitude: number | string
    longitude: number | string
    sourceLatitude?: number
    sourceLongitude?: number
    googleForceLatLon?: boolean
    googlePlaceId?: number
    title?: string
    app?: string
    dialogTitle: string
    dialogMessage: string
    cancelText: string
    appsWhiteList?: string[]
}
```

**Comment by tschoffelen** [comment]
I think this is a bug in the TypeScript definition not matching up to the actual code. Feel free to submit a PR to improve the `index.d.ts` file.

**Comment by tschoffelen** [comment]
Fixed in #51.
